spec/queries/content_items_query_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe ContentItemsQuery, type: :query do
```

Comment:
Hopefully addressed in https://github.com/alphagov/content-performance-manager/pull/72/commits/c5cb55df5dcd077bcb5370b85f123aa568e28134